logstash-core/logstash-core.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency 'puma', '~> 2.16', '>= 2.16.0'
   gem.add_runtime_dependency "jruby-openssl", "0.9.13" # Required to support TLSv1.2
   gem.add_runtime_dependency "chronic_duration", "0.10.6"
-  gem.add_runtime_dependency "jruby-monitoring", '~> 0.1'
+  gem.add_runtime_dependency "jruby-monitoring", '~> 0.3.1'
```

Comment:
I know it changes requirements, but is also good to be more strict to avoid confusions and errors in the feature.